configure.ac

```diff
@@ -145,7 +145,7 @@ case "$host" in
 		fi
 		HOST_CC="gcc"
 		# Windows Vista or later is required
```

Comment:
Can this comment be updated to reflect the changes?

Reply:
Yes good catch. I should have grepped for Vista. (and maybe  say "Windows7" instead of "Windows 7" for future greps)